src/pyfaf/actions/reposync.py

```diff
@@ -85,7 +85,7 @@ def run(self, cmdline, db):
                     try:
                         repo_instance = {
                             'instance' : repo_types[repo.type](repo.name,
-                                            [url.url for url in repo.url_list]),
+                                                               [url.url for url in repo.url_list]),
```

Comment:
BTW, any little change in length of the line above will devastate your efforts.
I woulr rather keep this line at it was and add move `repo.name` from line 87 to line 88 and align it with `[url.url for ...`.

Reply:
@jfilak  once fixed we can prevent any break using pep8speak. So any further change will not devastate this, because if you change the line above you have to change this line as well.